libconcurrent/includes/threadtools.h

```diff
@@ -25,17 +25,21 @@ int synchStartThreadsN(uint32_t nthreads, void *(*func)(void *), uint32_t uthrea
 
 /// @brief This function returns whenever all the created posix threads and fibers spawned by StartThreadsN
 /// have completed the execution. 
-/// @param nthreads The number of posix threads minus 1 that StartThreadsN spawned.
+/// @param nthreads The number of posix threads that StartThreadsN spawned.
 void synchJoinThreadsN(uint32_t nthreads);
 
 /// @brief This function sets the CPU affinity of the running thread to cpu_id, where cpu_id
 /// should be a unique integer in {0, ..., N-1}, where N is the amount of available procissing cores.
 int synchThreadPin(int32_t cpu_id);
 
-/// @brief This function returns the CPU affinity of the running thread. More specifically, it returns
-/// a unique integer in {0, ..., N-1}, where N is the amount of available procissing cores.
+/// @brief This function returns the id of the running thread (posix or fiber). More specifically, it returns
+/// a unique integer in {0, ..., N-1}, where N is the amount of the running threads.
```

Comment:
So, if we have `3` Posix threads, and for each we have `4` fiber threads, this function will return `{0, ...., 11}`, right?
Maybe you can also add this simple example here.

Reply:
Done.